priv/static/phoenix_html.js

```diff
@@ -7,12 +7,22 @@ function isLinkToSubmitParent(element) {
   return isLinkTag && shouldSubmitParent;
 }
 
+function getClosestForm(element) {
+  while (element && element !== document && element.nodeType === Node.ELEMENT_NODE) {
+    if (element.tagName.toLowerCase() === 'form') {
```

Comment:
We should use `element.tagName === 'FORM'` to follow the same pattern we used in the function above.